src/dyncfg/src/lib.rs

```diff
@@ -119,19 +119,6 @@ impl<T: ConfigType> Config<T> {
         &self.default
     }
 
-    /// Adds this config to the given set.
-    ///
-    /// Names are required to be unique within a set, but each set is entirely
```

Comment:
Wild copy-pasta here, presumably by me. Let's move this full rustdoc to [ConfigSet::add] so we don't lose the bit of additional detail

Reply:
Done.